### ISSUE TITLE
JUnit Output

### ISSUE THREAD

**Issue by strowi** [optiopay/klar#107]
Hi,

thx for klar! Nice alternative to the sometimes buggy clair-cli. Is there any chance for a junit-compatible output format? This could be parsed by many ci-systems for better output?

regards,
Roman

**Comment by hashmap** [comment]
@strowi good idea. It makes sense to introduce some abstract over output, currently we support 2 (text and json) both are hardcoded, let's make it more configurable and extandable

**Comment by jlamande** [comment]
@hashmap do you know if the current json output format is compatible with any analyzer tool ?
I'd like to make results of vulnerability scans more user-friendly by reusing some existing tool.
Typically integrate it into Jenkins.

**Comment by hashmap** [comment]
@jlamande afaik no, if you want to generate reports in jenkins etc junit format is the way to go
